config/initializers/audited_hacks.rb

```diff
@@ -78,6 +78,7 @@ module AuditedHacks
 
   module ClassMethods
     def audited(options = {})
+      options[:on] = %i[create update destroy]
```

Comment:
perhaps `||=`, or maybe btter use `except`?

Reply:
Yeah, I think it needs some changes. If the user provides some options, then we want to filter `:touch` out; but if the options come empty, then Audited will [initialize](https://github.com/collectiveidea/audited/blob/main/lib/audited/auditor.rb#L499) `options[:on]` to include `:touch`. So we can't leave the options empty neither. I'll fix this.

Reply:
Fixed in bf0d35d